src/sliceDataStorage.h

```diff
@@ -208,6 +210,18 @@ class SliceMeshStorage : public SettingsMessenger // passes on settings from a M
      * \return whether a particular extruder is used by this mesh on a particular layer
      */
     bool getExtruderIsUsed(int extruder_nr, int layer_nr) const;
+
+    /*!
+     * \return the mesh's user specified z seam hint
+     */
+    Point getZSeamOrigin() const {
```

Comment:
Code style: `{` on separate line.

I think the naming of this function is a bit weird. The word 'origin' doesn't seem the right word...
Maybe just `getZSeamLocationHint()` ?